nexus/src/config.rs

```diff
@@ -301,6 +391,8 @@ mod test {
             [updates]
             trusted_root = "/path/to/root.json"
             default_base_url = "http://example.invalid/"
+            [tunables]
+            max_vpc_ipv4_subnet_prefix = 26
```

Comment:
Maybe use a different value for this so that the test isn't working by accident (i.e., picking up the default somehow)?